submissions/enduranceIdisi/Javascript/product-price/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="script.js"></script>
```

Comment:
Its bad coding practice to have your script src in the head tags

Reply:
I have placed it in the body tag @ToluEnny24